client/components/confirmation-modal/styles.scss

```diff
@@ -6,7 +6,7 @@
 	}
 
 	&__separator {
-		margin: 24px -24px;
```

Comment:
Converted to use Gutenberg's units, so that next time we upgrade we're ok.
It looks like in some version of Gutenberg, the modal has `24px` padding. In other versions of Gutenberg it has `32px`.